resources/warnings.json

```
@@ -6137,7 +6137,7 @@
     "versions": [
       {
         "lastVersion": "2.0.4",
-        "pattern": ".*"
+        "pattern": "(^[012].*)"
```

```suggestion
        "pattern": "([12])(|[.-].*)"
```

Otherwise will match version `10`. Other changes for consistency.

---

Hi Daniel, 

Thanks for the updates. My RegEx capabilities were never great, so I appreciate you checking the string. 

Can you confirm this will work from your end? I'm quick RegEx101 test is unable to identify the entire string. Thanks!

---

> I'm quick RegEx101 test is unable to identify the entire string

The "match" view on that site seems to have `#find` semantics rather than `#matches`. The "unit test" view seems more suitable for this use case.

I confirm the format of regexes we've been using since about 2016 in this file works 😉